extensions/sat/README.md

```diff
@@ -11,24 +11,41 @@ This document explains the fields of the Satellite Extension to a STAC Item. Sat
 The Satellite extension requires the [Instrument Fields](../../item-spec/common-metadata.md#instrument).
 
 - [Example (Landsat 8)](examples/example-landsat8.json)
+- [Example (Sentinel 1)](examples/example-sentinel1.json)
 - [JSON Schema](json-schema/schema.json)
 
 ## Item fields
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
+| sat:platform_serial_identifier | string | The International Designator, also known as COSPAR ID, and NSSDCA ID |
```

Comment:
That seems indeed useful for search, but why is the name so different from all the names mentioned in the description?